app/api/attendees.py

```diff
@@ -74,6 +74,11 @@ def before_post(self, args, kwargs, data):
                 {'pointer': '/data/relationships/ticket'},
                 "Ticket belongs to a different Event"
             )
+
+        # populate event id and ticket id
+        data['ticket_id'] = ticket.id
+        data['event_id'] = ticket.event_id
```

Comment:
Not needed

Reply:
Hi @iamareebjamal I am trying to debug what is the exact reason of the `ticket_id` and `event_id`  becoming `null` after the object creation. That's why I tried passing them in the data payload. Any you help me find out what can be the exact reason? Am I going in the right direction as per tracing the problem?